src/ape/managers/config.py

```diff
@@ -86,6 +87,9 @@ class ConfigManager(BaseInterfaceModel):
     version: str = ""
     """The project's version."""
 
+    meta: PackageMeta = PackageMeta
```

Comment:
Type error here -- you are not initializing the default value
Perhaps you meant to do `meta: PackageMeta = PackageMeta()` ?

Reply:
Agree, please change to
```suggestion
    meta: PackageMeta = PackageMeta()
```